draft-ietf-quic-transport.md

```diff
@@ -5753,13 +5753,184 @@ DecodePacketNumber(largest_pn, truncated_pn, pn_nbits):
    return candidate_pn
 ~~~
 
+# Overview of Security Properties {#security-properties}
```

Comment:
This is probably better placed as a "Security Considerations" section, with the current text incorporated into this as appropriate (mostly under mitigations).

Reply:
Two thoughts here: 
- This was an appendix since TLS 1.3 did it as an appendix. It's kind of nice that way, since we're documenting what we believe the rest of the document says, not introducing any new attacks or mitigations that shouldn't already have been covered above.
- From a purely aesthetic perspective, having 21.11.3.2. as a section means that I refer to one of these items as: 
> Section 21.11.3.2., point 3.

All that said, I've moved this to the last section in Security Considerations for the time being, and I don't have a super strong opinion on where it finally lands. :)
TLS is also a little bit different in that most of the document is a security consideration in one way or another, so some of the reasoning that resulted in structure there is not necessarily reflected in QUIC.

Reply:
It seems worth excerpting RFC 3552:
> While it is not a requirement that any given protocol or system be immune to all forms of attack, it is still necessary for authors to consider as many forms as possible.  Part of the purpose of the Security Considerations section is to explain what attacks are out of scope and what countermeasures can be applied to defend against them.
>
> There should be a clear description of the kinds of threats on the described protocol or technology.  This should be approached as an effort to perform "due diligence" in describing all known or foreseeable risks and threats to potential implementers and users.
>
> Authors MUST describe
>
>    1.   which attacks are out of scope (and why!)
>    2.   which attacks are in-scope
>          1.  and the protocol is susceptible to
>          2.  and the protocol protects against
>
> At least the following forms of attack MUST be considered:  eavesdropping, replay, message insertion, deletion, modification, and man-in-the-middle.  Potential denial of service attacks MUST be identified as well.  If the protocol incorporates cryptographic protection mechanisms, it should be clearly indicated which portions of the data are protected and what the protections are (i.e., integrity only, confidentiality, and/or endpoint authentication, etc.).  Some indication should also be given to what sorts of attacks the cryptographic protection is susceptible.  Data which should be held secret (keying material, random seeds, etc.) should be clearly labeled.

In other words, this reads a lot like a Security Considerations section.

Reply:
Thanks! I've moved it up and updated the text to refer to it as such.